<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateManager</class>
 <widget class="QWidget" name="TemplateManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>422</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Repository URL</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="repoUrl"/>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="updateRepository">
        <property name="icon">
         <iconset theme="view-refresh"/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="selectAll">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="edit-select-all"/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="unselectAll">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="deletecell"/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="downloadFromRepo">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="edit-download"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QProgressBar" name="totalProgressBar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="widgetList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
